src/main/java/pl/javastart/jjdind49_katibar_zad20_1/UserController.java

```diff
@@ -0,0 +1,57 @@
+package pl.javastart.jjdind49_katibar_zad20_1;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.List;
+
+@Controller
+public class UserController {
+
+    private final UsersRepo usersRepo;
+
+    public UserController(UsersRepo usersRepo) {
+        this.usersRepo = usersRepo;
+    }
+
+    @RequestMapping("/repo")
+    public String startPOST(@RequestParam String firstName, @RequestParam String lastName, @RequestParam Integer age) {
+        if (firstName != null) {
+            createNewUser(firstName, lastName, age);
+            return "redirect:/success.html";
+        } else {
+            return "redirect:/err.html";
+        }
+    }
+
+    @RequestMapping("/users")
+    @ResponseBody
+    public String showUsers() {
+        List<User> userList = usersRepo.getUserList();
+        String result = "";
+        for (User user : userList) {
+            result += user.toString() + "<br/>";
+        }
+        return result;
+    }
+
+    @GetMapping("/add")
+    public String startGET(@RequestParam(value = "imie", required = false) String firstName,
+                           @RequestParam("nazwisko") String lastName,
+                           @RequestParam("wiek") Integer age) {
+        if (firstName != null) {
+            createNewUser(firstName, lastName, age);
+            return "/success.html";
+        } else {
+            return "/err.html";
+        }
+    }
```

Comment:
- nazwa startGETnie jest zapisana jako camelCase
- nazwa startGET niewiele mówi, może `addUser` byłoby lepsze?
- to robi dokładnie to samo co `startPOST`, zostaw tylko jedną z tych dwóch metod

Reply:
Pozmieniałam na razie tylko nazwy metod.
Nie mogę poradzić sobie, żeby z 2 podobnych metod zrobić jedną. Próbowałam dodać dwie metody, jako method = {RequestMethod.POST, RequestMethod.GET}, żeby jedna metoda obsługiwała obydwa rodzaje, ale niestety ciągle wyrzuca tylko Whitelabel Error Page.

Masz może jakąś wskazówkę, jak to można rozgryźć?